.circleci/config.yml

```diff
@@ -60,11 +60,11 @@ jobs:
             aws configure set region $AWS_REGION
             aws configure set default.output json
             aws configure list  # Get confirmation it worked in your logs
-            aws lambda update-function-code --function-name coreRebuild --zip-file fileb://lambda.zip
+            aws lambda update-function-code --function-name triggerRebuild --zip-file fileb://lambda.zip
```

Comment:
@aaron-collier I believe this change will require updates in AWS, namely creating a new lambda named `triggerRebuild` (since we apparently can't rename an AWS lambda function?). Is that true? How annoying is this change? If too annoying, I can roll this delta back.